services-custom/s3-transfer-manager/src/main/java/software/amazon/awssdk/transfer/s3/model/FileUpload.java

```diff
@@ -32,12 +30,6 @@ public interface FileUpload extends ObjectTransfer {
      * <p>
      * The information object is serializable for persistent storage until it should be resumed.
      * See {@link ResumableFileUpload} for supported formats.
-     * 
-     * <p>
-     * Currently, it's only supported if the underlying {@link S3AsyncClient} is CRT-based (created via
```

Comment:
Can we update the javadoc to mention it's only supported if the underlying {@link S3AsyncClient} is CRT-based... OR the underlying  {@link S3AsyncClient} has multipart configuration enabled?